Makefile

```diff
@@ -5,11 +5,22 @@ verify:
 	@# Use "-x" if you want to skip external links
 	@tools/verify-links.sh -t -v .
 	@echo Running the spec phrase checker:
-	@tools/verify-specs.sh -v spec.md documented-extensions.md json-format.md \
-		http-protocol-binding.md http-webhook.md mqtt-protocol-binding.md \
-		nats-protocol-binding.md kafka-protocol-binding.md avro-format.md \
+	@tools/verify-specs.sh -v \
+		documented-extensions.md \
```

Comment:
should alphabetize this list (or would it be better to just use "*.md"?

Reply:
doit - they were supposed to be :-)
fixed

Reply:
good catch